# geometry_msgs/Point chest_position
std_msgs/Float64 dist_chest
std_msgs/Float64 threshold